### DISCUSSION TITLE
Is there a Windows version of the showcase app?

### DISCUSSION THREAD

**Discussion by stamminator** [bdlukaa/fluent_ui#1096] [General]
The [web showcase app](https://bdlukaa.github.io/fluent_ui/) is great, but is there also an app that showcases the capabilities of Windows-native apps, similar to the [WinUI 3 Gallery](https://github.com/microsoft/WinUI-Gallery) app or [React Native Gallery](https://github.com/microsoft/react-native-gallery)?

**Comment by bdlukaa** [comment]
This is off the scope of the project.

You can download the code and test it on your Windows device.
